src/storage/client/StorageClient.cpp

```diff
@@ -235,6 +235,88 @@ folly::SemiFuture<StorageRpcResponse<cpp2::EdgePropResponse>> StorageClient::get
 }
 
 
+folly::Future<StatusOr<cpp2::EdgeKeyResponse>> StorageClient::getEdgeKeys(
+    GraphSpaceID space,
+    VertexID vid,
+    folly::EventBase* evb) {
+    std::pair<HostAddr, cpp2::EdgeKeyRequest> request;
+    PartitionID part = partId(space, vid);
+    auto partMeta = getPartMeta(space, part);
+    CHECK_GT(partMeta.peers_.size(), 0U);
+    const auto& leader = this->leader(partMeta);
+    request.first = std::move(leader);
```

Comment:
leader is const auto &, maybe can not move.

Reply:
Good catch